LayoutTests/imported/w3c/web-platform-tests/cookie-store/change_eventhandler_for_document_cookie.https.window-expected.txt

```diff
@@ -2,5 +2,5 @@
 PASS document.cookie set/overwrite/delete observed by CookieStore
 PASS CookieStore set/overwrite/delete observed by document.cookie
 FAIL CookieStore agrees with document.cookie on encoding non-ASCII cookies assert_equals: Cookie we wrote using document.cookie in cookie jar expected (string) "DOCUMENT-🍪=🔵" but got (undefined) undefined
-PASS document.cookie agrees with CookieStore on encoding non-ASCII cookies
+FAIL document.cookie agrees with CookieStore on encoding non-ASCII cookies assert_equals: Cookie we wrote using CookieStore in document.cookie expected (string) "DOCUMENT-🍪=🔵" but got (undefined) undefined
```

Comment:
I don't think this patch should cause new failures?

Reply:
Fixed!